### ISSUE TITLE
"filePath" absolute path in SD version 3.0

### ISSUE THREAD

**Issue by didoo** [amzn/style-dictionary#498]
I have just updated SD to version 3.0 (rc1) and everything went perfectly smooth.
One thing I have noticed though is that the new "filePath" attribute has an absolute path:

```
 {
    "value": "#767676",
    "type": "color",
    "group": "mono",
    "filePath": "/Users/cristianorastelli/src/design-system/tokens/src/badoo/products/shared/color/color-mono.json",
    "isSource": true,
    "original": {
      "value": "#767676",
      "type": "color",
      "group": "mono"
    },
```

In our specific case, we're using a custom format that "dumps" the entire structure in a JSON file that it's used in the style guide website, and we can easily remove the "filePath" property from the output, so no big deal per se.

The question I have is: in general, why not use a relative path for this attribute? What is the benefit of having an absolute path for this? (it may cause differences when run on different machines, while the actual tokens remain the same).

**Comment by dbanksdesign** [comment]
Makes total sense, opened a PR to fix this. Now `filePath` will be relative (or absolute if provided an absolute path in the configuration). Thanks!

**Comment by chazzmoney** [comment]
Is fixed in code and will be part of 3.0.0-rc.2

Thanks @didoo and @dbanksdesign for finding and fixing respectively!
